samples/csharp_dotnetcore/50.teams-messaging-extensions-search/Resources/connectorCard.json

```diff
@@ -0,0 +1,38 @@
+{
+  "@context": "https://schema.org/extensions",
+  "@type": "MessageCard",
+  "sections": [
+    {
+      "activityTitle": "Steve Tweeted",
+      "activitySubtitle": "With #MicrosoftTeams",
+      "activityImage": "http://connectorsdemo.azurewebsites.net/images/MSC12_Oscar_002.jpg"
```

Comment:
Should this be templated, and pointing to a file in the deployed project?
MSC12_Oscar_002.jpg

Reply:
Updated with the deployed URL